eng/packages/http-client-csharp-mgmt/generator/Azure.Generator.Management/src/Providers/OperationMethodProviders/ResourceOperationMethodProvider.cs

```diff
@@ -315,46 +320,5 @@ protected virtual IReadOnlyList<MethodBodyStatement> BuildReturnStatements(Value
 
             return statements;
         }
-
-        // TODO -- we should be able to just use the parameters from convenience method. But currently the xml doc provider has some bug that we build the parameters prematurely.
```

Comment:
think we should keep the comment in the new place as well, so that we know we should fix it later